<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Włącz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="oknoB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Zezwalaj na pokazywanie w okienku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sngTB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Odczytuj tekst piosenki</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="msB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Ilość pokazywanych próbek w ciągu określonego czasu (ms)</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="lpB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pokaż lewy i prawy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pokaż próbki dźwięku</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="refB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Szybkość odświeżania (ms)</string>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>38</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>88</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pokaż paski</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pokaż kreski</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kolor2B">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pokaż w wielu kolorach</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoFillBackgroundB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Automatycznie wypełniaj tło</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>oknoB</tabstop>
  <tabstop>sngTB</tabstop>
  <tabstop>sB</tabstop>
  <tabstop>lpB</tabstop>
  <tabstop>pB</tabstop>
  <tabstop>kB</tabstop>
  <tabstop>kolor2B</tabstop>
  <tabstop>autoFillBackgroundB</tabstop>
  <tabstop>refB</tabstop>
  <tabstop>msB</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
